client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -120,81 +127,93 @@ const NewsletterSetup: Step = ( { navigation } ) => {
 	};
 
 	const stepContent = (
-		<>
```

Comment:
Yuck. It looks as if I touched every line in this block. All I did was remove the unnecessary fragment which resulted in one tab reduction for all encapsulated lines. I can revert this change if one prefers I did not do that.

Reply:
In these cases, I recommend to the reviewer to ignore whitespace. add `?w=1` to the URL. Not a problem.

Reply:
Omg... 😱  I never knew this. This is really useful. Thanks!